app/controllers/estimates_controller.rb

```diff
@@ -44,8 +44,12 @@ def download
         left: "1cm",
         right: "1cm",
       },
+      timezone: "Europe/London",
+      vision_deficiency: "deuteranopia",
```

Comment:
The deuteranopia bit still needs to go, I think?

Reply:
Fair enough - I guess I was trying to future proof it. But the document is in Black & White so shouldn't affect people who are colour blind.

Reply:
Ohh, I see. Yeah, in practice it doesn't matter either way. If we were to have kept it in it would have needed some explanation, because I had to look up `deuteranopia` and then try to work out what effect passing it in in config would have on the output. 

If ever we start producing colour PDFs I think that's something we can worry about then (probably discussions needed as to whether we optimise for deuteranopia vs protanopia etc...)